test/common/fit_common.py

```diff
@@ -777,7 +779,7 @@ def power_control_all_nodes(state):
 
 
 def mongo_reset():
-    # clears the Mongo database on ORA to default, returns 0 if successful
+    # clears the Mongo database on host to default, returns 0 if successful
```

Comment:
@larry-dean  I have another story to fix this routine. Its broken and not used so let's leave it for now.